### ISSUE TITLE
Click on article on landing page --> directs to article on map

### ISSUE THREAD

**Issue by SDC-LZC** [Allied-Voices/alliedvoices#228]
Need unique url for each article

**Comment by Nyette** [comment]
I don't think that this functionality is possible in the app's current state. Whenever the map page renders, it will always show the first page and then allow the user to proceed from there. You cannot jump to a specific page to access a specific article from the get go because neither have rendered. On top of that, the page associated with each article will change as articles are added. We would have to change a lot about the map page for this to work.
